net-misc/spice-gtk/spice-gtk-0.42-r4.ebuild

```diff
@@ -40,8 +42,8 @@ RDEPEND="
 	sys-libs/zlib
 	>=x11-libs/cairo-1.2
 	>=x11-libs/pixman-0.17.7
-	x11-libs/libX11
-	gtk3? ( x11-libs/gtk+:3[introspection?] )
+	X? ( x11-libs/libX11 )
+	gtk3? ( x11-libs/gtk+:3[X?,wayland?,introspection?] )
```

Comment:
This should be revbumped so that people actually rebuild it with the relevant changes.